src/components/InventoryHostStaleness/InventoryHostStaleness.js

```diff
@@ -2,9 +2,21 @@
 import React from 'react';
 import HostStalenessCard from './HostStalenessCard';
 import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
-import { GENERAL_HOST_STALENESS_WRITE_PERMISSION } from './constants';
+import {
+  GENERAL_HOST_STALENESS_READ_PERMISSION,
+  GENERAL_HOST_STALENESS_WRITE_PERMISSION,
+} from './constants';
+import {
+  GENERAL_HOSTS_READ_PERMISSIONS,
+  GENERAL_HOSTS_WRITE_PERMISSIONS,
+} from '../../constants';
 
-const REQUIRED_PERMISSIONS = [GENERAL_HOST_STALENESS_WRITE_PERMISSION];
+const REQUIRED_PERMISSIONS = [
+  GENERAL_HOST_STALENESS_WRITE_PERMISSION,
+  GENERAL_HOST_STALENESS_READ_PERMISSION,
+  GENERAL_HOSTS_READ_PERMISSIONS,
+  GENERAL_HOSTS_WRITE_PERMISSIONS,
+];
 
 const InventoryHostStaleness = () => {
   const { hasAccess: canModifyHostStaleness } =
```

Comment:
You have to set `checkAll` (the second parameter): so that you make the logical AND operation and check that all of the permissions are fulfilled at a time